packages/website/src/components/DocContainer.tsx

```diff
@@ -21,14 +21,14 @@ export interface DocContainerProps {
 export function DocContainer({ name, kind, excerpt, summary, typeParams, children, extendsTokens }: DocContainerProps) {
 	return (
 		<>
-			<div className="bg-white dark:bg-dark border-b-solid border-gray border-0.5 border-width-0.5 sticky top-0 px-10 py-2">
```

Comment:
Why remove all the borders 🤔

Reply:
You really think? It looks more unfinished than before now 🙈

Reply:
Like

Before:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/20760160/180806734-e99caa99-f8f3-46eb-ba77-8a09baf20839.png">

After:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/20760160/180806645-a46c5598-f4ee-4520-bf68-3bdbf7c1cb0c.png">

Reply:
Beside maybe the bottom border on the top items the other borders feel mostly unnecessary to me because the separation is clear.

Most of the docs sites I've looked at don't add separators like this:

Motion One:
<img width="1424" alt="Screen Shot 2022-07-25 at 10 57 31 AM" src="https://user-images.githubusercontent.com/77477100/180808326-7f223319-47e8-4247-870e-3b50b4edf478.png">

TailwindCSS:
<img width="1534" alt="Screen Shot 2022-07-25 at 10 56 08 AM" src="https://user-images.githubusercontent.com/77477100/180808406-293a8845-58f6-40b7-9cf9-1c1d4fa7f87b.png">

Next.JS:
<img width="1216" alt="Screen Shot 2022-07-25 at 10 56 43 AM" src="https://user-images.githubusercontent.com/77477100/180808499-1d882156-9f0c-4d3c-9821-0dfbea4def9a.png">

Reply:
Sadly, all your screenshots are done on OSX which hides scrollbars by default. Which is also what I assumed why it might look "better" on your end.

Reply:
The way the current site is setup, I'd rather keep the borders still until we find a better way, because just removing them without some layout shifting is making the site look like ass for now.